benchmark/index.js

```diff
@@ -165,6 +162,128 @@ test('migration (alone)', async (t) => {
   await ended.promise
 })
 
+test('migrate (+db1 +db2)', async (t) => {
+  rimraf.sync(db2Path)
+  t.pass('delete db2 folder to start clean')
+
+  const keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+  const sbot = SecretStack({ appKey: caps.shs })
+    .use(require('ssb-db'))
+    .use(require('../'))
+    .call(null, { keys, path: dir })
+
+  await sleep(500) // some silence to make it easier to read the CPU profiler
+
+  const ended = DeferredPromise()
+  const start = Date.now()
+  sbot.db2migrate.start()
+
+  pull(
+    fromEvent('ssb:db2:migrate:progress', sbot),
+    pull.filter((progress) => progress === 1),
+    pull.take(1),
+    pull.drain(async () => {
+      const duration = Date.now() - start
+      t.pass(`duration: ${duration}ms`)
+      fs.appendFileSync(reportPath, `| Migrate (+db1 +db2) | ${duration}ms |\n`)
+      await sleep(4000) // wait for new log FS writes to finalize
```

Comment:
can you do a sbot.db.onDrain here?

Reply:
Oh yeah, good idea, I was copied this `await sleep(4000)` from previous tests above, but those didn't have db2 for me to call sbot.db.onDrain. I'll do that here